<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../qrs/resources.qrc">
    <normaloff>:/Staccato.ico</normaloff>:/Staccato.ico</iconset>
  </property>
  <property name="locale">
   <locale language="Portuguese" country="Brazil"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_26">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="styleSheet">
       <string notr="true">QTabBar::tab:disabled { width: 0; height: 0; margin: 0; padding: 0; border: none; }
</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabOrcamentos">
       <attribute name="title">
        <string>Orçamentos</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="WidgetOrcamento" name="widgetOrcamento" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabVendas">
       <attribute name="title">
        <string>Vendas</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_20">
        <item row="0" column="0">
         <widget class="WidgetVenda" name="widgetVenda" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabCompras">
       <attribute name="title">
        <string>Compras</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_21">
        <item row="0" column="0">
         <widget class="WidgetCompra" name="widgetCompra" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabEntregas">
       <attribute name="title">
        <string>Logística</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="WidgetLogistica" name="widgetLogistica" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabNFe">
       <attribute name="title">
        <string>NFe</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_17">
        <item row="0" column="1">
         <widget class="WidgetNfe" name="widgetNfe" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabEstoque">
       <attribute name="title">
        <string>Estoque</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_22">
        <item row="0" column="0">
         <widget class="WidgetEstoque" name="widgetEstoque" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabContas">
       <attribute name="title">
        <string>Contas</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_36">
        <item row="0" column="0">
         <widget class="WidgetConta" name="widgetConta" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>764</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOp_es">
    <property name="title">
     <string>Opções</string>
    </property>
    <widget class="QMenu" name="menuTema">
     <property name="title">
      <string>Tema</string>
     </property>
     <addaction name="actionClaro"/>
     <addaction name="actionEscuro"/>
    </widget>
    <addaction name="actionImportaProdutos"/>
    <addaction name="menuTema"/>
    <addaction name="actionConfigura_es"/>
   </widget>
   <widget class="QMenu" name="menuAdministra_o">
    <property name="title">
     <string>Administração</string>
    </property>
    <addaction name="actionCadastrarProfissional"/>
    <addaction name="actionGerenciar_Lojas"/>
    <addaction name="actionGerenciar_Transportadoras"/>
    <addaction name="actionCadastrarUsuario"/>
    <addaction name="actionCadastrarFornecedor"/>
    <addaction name="actionCadastrarProdutos"/>
   </widget>
   <widget class="QMenu" name="menuVenda">
    <property name="title">
     <string>Venda</string>
    </property>
    <addaction name="actionCriarOrcamento"/>
    <addaction name="actionCadastrarCliente"/>
   </widget>
   <widget class="QMenu" name="menuAjuda">
    <property name="title">
     <string>Ajuda</string>
    </property>
    <addaction name="actionSobre"/>
   </widget>
   <addaction name="menuVenda"/>
   <addaction name="menuAdministra_o"/>
   <addaction name="menuOp_es"/>
   <addaction name="menuAjuda"/>
  </widget>
  <action name="actionCriarOrcamento">
   <property name="text">
    <string>Criar orçamento</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionCadastrarProdutos">
   <property name="text">
    <string>Gerenciar produtos</string>
   </property>
  </action>
  <action name="actionCadastrarCliente">
   <property name="text">
    <string>Gerenciar cliente</string>
   </property>
   <property name="toolTip">
    <string>Cadastrar cliente</string>
   </property>
  </action>
  <action name="actionCadastrarUsuario">
   <property name="text">
    <string>Gerenciar usuários</string>
   </property>
   <property name="toolTip">
    <string>Cadastrar usuário</string>
   </property>
  </action>
  <action name="actionCadastrarProfissional">
   <property name="text">
    <string>Gerenciar profissionais</string>
   </property>
   <property name="toolTip">
    <string>Cadastrar profissional</string>
   </property>
  </action>
  <action name="actionGerenciar_Transportadoras">
   <property name="text">
    <string>Gerenciar transportadoras</string>
   </property>
  </action>
  <action name="actionGerenciar_Lojas">
   <property name="text">
    <string>Gerenciar lojas</string>
   </property>
  </action>
  <action name="actionCadastrarFornecedor">
   <property name="text">
    <string>Gerenciar fornecedores</string>
   </property>
   <property name="toolTip">
    <string>Cadastrar fornecedor</string>
   </property>
  </action>
  <action name="actionImportaProdutos">
   <property name="text">
    <string>Importar tabela fornecedor</string>
   </property>
  </action>
  <action name="actionSobre">
   <property name="text">
    <string>Sobre o ERP Staccato</string>
   </property>
  </action>
  <action name="actionClaro">
   <property name="text">
    <string>Claro</string>
   </property>
  </action>
  <action name="actionEscuro">
   <property name="text">
    <string>Escuro</string>
   </property>
  </action>
  <action name="actionConfigura_es">
   <property name="text">
    <string>Configurações</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>WidgetOrcamento</class>
   <extends>QWidget</extends>
   <header>src/widgetorcamento.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetVenda</class>
   <extends>QWidget</extends>
   <header>src/widgetvenda.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetCompra</class>
   <extends>QWidget</extends>
   <header>src/widgetcompra.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetLogistica</class>
   <extends>QWidget</extends>
   <header>src/widgetlogistica.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetNfe</class>
   <extends>QWidget</extends>
   <header>src/widgetnfe.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetEstoque</class>
   <extends>QWidget</extends>
   <header>src/widgetestoque.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetConta</class>
   <extends>QWidget</extends>
   <header>src/widgetconta.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qrs/resources.qrc"/>
 </resources>
 <connections/>
</ui>
